### DISCUSSION TITLE
Rationales for starting a new repo?

### DISCUSSION THREAD

**Discussion by chuzhe-as-a-dev** [dotnet/eShop#118] [General]
Is there any material on why we switched to a new repo and on the difference from the old one? Any pointer/reference is appreciated!

**Comment by Eric-TPS** [comment]
Yep, I've noticed that this repository seems primarily configured for Microsoft's Azure Service Fabric, rather than Docker Host. Key elements like Dockerfiles and build instructions are missing, which suggests a shift away from industry-standard Docker tools. This change appears to require a reliance on more Microsoft-centric tools. Which leads me to believe that this might be a strategic move by Microsoft to promote their own ecosystem over commonly used industry technologies.
